src/praisonai/tests/unit/test_decorator_enforcement.py

```
@@ -1,55 +1,49 @@
         
         try:
-            # This should now be blocked by the decorator
-            result = shell_tools.execute_command('echo "This should be denied"')
-            print('❌ Command executed when it should have been denied!')
-            return False
-        except PermissionError as e:
-            print(f'✅ Decorator enforcement working: {e}')
-            return True
+            result = test_function("dangerous command")
+            print("❌ Command executed when it should have been denied!")
+            assert False, "Command executed when it should have been denied!"
```

Ruff (0.11.9)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)